package.json

```diff
@@ -3,13 +3,14 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "typescript": "index.ts",
   "dependencies": {},
   "devDependencies": {
+    "@types/lodash": "4.14.136",
     "@types/react-dom": "16.0.11",
     "eslint": "^5.10.0",
     "eslint-config-tarwich": "github:tarwich/eslint-config-tarwich",
-    "mobx": "5.6.0",
-    "mobx-react": "^5.4.3",
+    "lodash": "4.17.14",
```

Comment:
Have you done an `npm link` and made sure this runs in our project?

Reply:
yes i am testing it and running it in our project